src/PhpOption/Option.php

```diff
@@ -158,20 +160,32 @@ public static function ensure($value, $noneValue = null)
     public static function lift($callback, $noneValue = null)
     {
         return function () use ($callback, $noneValue) {
+            /** @psalm-var list<mixed> */
```

Comment:
I don't use psalm, so this can be removed. phpstan doesn't read these annotations, I think?

Reply:
When I talked to Ondřej in October he said he planned to add support for `list` at some point, but for now that can just be `array<mixed>`, which means the psalm prefix can go

Reply:
So, this line should be deleted, since phpstan can't understand it?

Reply:
I doubt that PHPStan can understand `@psalm-...` annotations. Maybe there's `@phpstan-var`?